### ISSUE TITLE
(BUG?) Local Media Folder Scraper overrides any Account priority and result sorting.

### ISSUE THREAD

**Issue by tundrablasen** [Tikipeter/tikipeter.github.io#171]
Hey Tiki,

i noticed following behavior:

I have following Accounts active:
1. Real-Debrid - Priority: 1 - check RD cloud first: off
2. Local Folder Scraper - ignore filter and size: off - check local folder first: off

**my expectation** 
would be, that RD + local media folder gets scrapped in parallel and depending on results sorting the best source from RD or Media Folder gets picked.

**actual bevaviour**
is though, that Media Folder Scraper is getting checked FIRST (before even checking RD) and whenever ANY file is found it gets picked and played immediately. RD is not even scrapped. it could be a 720p release, even though I requested in results sorting that I prefer 4K with Dolby Vision (and that would be available in RD).

https://paste.kodi.tv/ogerohuxic

Settings screenshots:
![IMG_9115](https://github.com/user-attachments/assets/81252675-db5e-47a4-92f9-5fb380490732)
![IMG_9114](https://github.com/user-attachments/assets/49a75231-af43-412e-a216-c496b11dca40)

**Tl;dr
As soon as you turn on Local Media Folder Scraping, Local Media gets checked first and a Source gets picked before checking RD or any other Account. even if you said explicitly in the settings you want to have a another prioritization**

**Comment by Tikipeter** [comment]
If you have a local file available then it will be given priority.

**Comment by tundrablasen** [comment]
is this a technical limitation? if not I think it would be better to change that. you can also give priority to local files with the button "put local files to the top".

for example in my use case I auto download lower quality files because I don't have so much local space. but if I have internet available I rather watch higher quality streams.
